### DISCUSSION TITLE
Trying to download DeDRM_tools_7.1.0.zip

### DISCUSSION THREAD

**Discussion by rxmom03** [apprenticeharper/DeDRM_tools#1578] [Q&A] [answered]
I am having the same problem as a previous poster - I have just downloaded Calibre 5.12 and am trying to update my plugins with DeDRM 7.1 I am not unzipping the file. Thanks for any help you can give! 
Here is the error I am getting - 

calibre, version 5.12.0
ERROR: Unhandled exception: <b>InvalidPlugin</b>:The plugin in 'C:\\Users\\maryb\\Downloads\\DeDRM_tools_7.1.0.zip' is invalid. It does not contain a top-level __init__.py file

calibre 5.12 [64bit]  embedded-python: True is64bit: True
Windows-10-10.0.19041 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19041')
Python 3.8.5
Windows: ('10', '10.0.19041', '', 'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: EpubMerge (2, 11, 0) && EpubSplit (3, 0, 0) && FanFicFare (4, 0, 2) && Kobo Utilities (2, 12, 3) && KoboTouchExtended (3, 2, 13)
Traceback (most recent call last):
  File "calibre\gui2\preferences\plugins.py", line 316, in add_plugin
  File "calibre\customize\ui.py", line 472, in add_plugin
  File "calibre\customize\ui.py", line 61, in load_plugin
  File "calibre\customize\zipplugin.py", line 284, in load
  File "calibre\customize\zipplugin.py", line 379, in _locate_code
calibre.customize.InvalidPlugin: The plugin in 'C:\\Users\\maryb\\Downloads\\DeDRM_tools_7.1.0.zip' is invalid. It does not contain a top-level __init__.py file

**Comment by ElleKayEm** [comment, marked as answer]
Unzip DeDRM_tools_7.1.0.zip.  Load DeDRM_plugin.zip.

**Reply by rxmom03** [reply]
Thank you! It has been so long ago since I had last done this I had forgotten all the steps.
